docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -1,113 +1,331 @@
 
-The `getSessionSigs()` function will try to create a session key for you and store it in local storage. You can also generate the session key yourself using `generateSessionKeyPair()` function and store it however you like. You can then pass the generated session key to `getSessionSigs()` as the `sessionKey` param.
+### `resourceAbilityRequests`
 
-In the example above, we construct a SIWE message manually, but you can use the `checkAndSignAuthMessage` when in a browser context.
+As mentioned previously, [Lit Resources and Abilities](./resources-and-abilities.md) are used to define what our Session Signature is allowing our Session Keys to do on the Lit network.
 
-## Resources You Can Request
+There are two parts to each ability request:
 
-You can pass an array of "resource ability requests" to the `getSessionSigs()` function, which will be presented to the user in the SIWE message - read more [here](resources-and-abilities) about Lit resources and abilities. The resources and abilities requested by the session key must be narrower or equal to the capabilities granted to it per the session capability object specified in the inner `AuthSig`. 
+#### `resource`
 
-When session capability objects are omitted from the `getSessionSigs()` function call, the SDK will generate a session capability object with **wildcard permissions against all of the resources in that category by default**, i.e. ability to perform operations against all access control conditions. Read more [here](capability-objects) about how to create custom session capability objects.
+This property is defining what Lit Resource we're granting the `ability` for. Each Lit Resource takes a resource identifier as an argument. You can pass `"*"` like so:
 
-## Clearing Local Storage
+```ts
+new LitPKPResource("*")
+```
 
-If you want to clear the session key stored in the browser local storage, you can call the [`disconnectWeb3` method](https://js-sdk.litprotocol.com/functions/auth_browser_src.ethConnect.disconnectWeb3.html).
+to grant an `ability` to all resources owned by the Ethereum address that authorized the request to generate Session Signatures for the Session Keys. In the case of this example, that would be the address corresponding to our `ETHEREUM_PRIVATE_KEY` constant (that we created an Ethers.js wallet for).
+
+Or you can specify a specific identifier for each resource as shown in the code snippets below.
+
+The available Lit Resource options are:
+
+##### `LitAccessControlConditionResource`
+
+Imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitAccessControlConditionResource } from "@lit-protocol/auth-helpers";
+```
+
+This resource specifies a Lit Access Control Conditions (ACC) the `ability` is being granted for. Specifying `"*"` grants the `ability` to any ACC, regardless of what the conditions are. To grant the `ability` for a specific ACC, you can specify the hashed key value of the ACC in place of `"*"` like so:
```

Comment:
@joshLong145 @Ansonhkg How do I get the hashed key value of the ACC to give to `LitAccessControlConditionResource` when setting `resourceAbilityRequests` for a Session Sig?

Reply:
in the `LitAccessControlConditionResource` class there's a public static method called `generateResourceString` which `Composes a resource string by hashing access control conditions and appending a data hash.`

So:
```ts
const accsResourceString =
  await LitAccessControlConditionResource.generateResourceString(
    accs,
    encryptRes.dataToEncryptHash
  );

const resourceAbilityRequests = [
{
        resource: new LitAccessControlConditionResource(accsResourceString),
        ability: LitAbility.AccessControlConditionDecryption,
}];
```